routers/api/v1/user/gpg_key.go

```diff
@@ -177,6 +178,12 @@ func VerifyUserGPGKey(ctx *context.APIContext) {
 	token := asymkey_model.VerificationToken(ctx.Doer, 1)
 	lastToken := asymkey_model.VerificationToken(ctx.Doer, 0)
 
+	form.KeyID = strings.TrimLeft(form.KeyID, "0")
```

Comment:
Can keys start with zeros, or is a leading zero always seen as padding?
Because if it is the former, this will probably produce a lot of problems in the future…

Reply:
Take a look at pubkey.KeyIDString()

The trimming is to fix potential issues relating to that